Contributions.md

```diff
@@ -0,0 +1,66 @@
+# Contributions
+
+This document list current and past contributions made to standards that has some relationship with the jQuery Foundation projects.
+
+# W3C
+
+## Pointer Events
+
+- [Recommendation 24 February 2015](https://www.w3.org/TR/pointerevents/)
+- [Editor's draft](https://w3c.github.io/pointerevents/)
+- participants: @scottgonzalez
+
+## Touch Events
+
+- [Recommendation 10 October 2013](https://www.w3.org/TR/touch-events/)
+- [Editor's draft](https://w3c.github.io/touch-events/)
+- participants: @scottgonzalez
+
+## Other contributions
+
+- [fixed the behavior](https://lists.w3.org/Archives/Public/public-whatwg-archive/2009Oct/0057.html) of `stepUp()` and `stepDown()` for `<input type="number">` based on the behavior of the spinner widget in __jQuery UI__. (from @scottgonzalez).
+
+# ECMAScript / TC39
+
+## Current proposals:
+
+| Proposal | Champion | Stage |
+|----------|----------|-------|
+| [String padding](https://github.com/tc39/proposal-string-pad-start-end) | Jordan Harband & Rick Waldron | 3 |
```

Comment:
It's more like I originally brought forward a proposal, but Jordan really gets the credit here. Would be nice if that was clearly stated